pkgs/top-level/all-packages.nix

```diff
@@ -18806,6 +18806,10 @@ with pkgs;
     pythonBindings = true;
   });
 
+  sublimeMergePackages = recurseIntoAttrs (callPackage ../development/tools/sublime-merge/packages.nix { });
+
+  sublimeMerge = sublimeMergePackages.sublimeMerge;
```

Comment:
Why not use `inherit`?

Reply:
fixing other comments, I'll explain this one though - this is a copy of the sublime text 3 package with as minimal changes as necessary to get it working.